fds/run.py

```diff
@@ -55,17 +54,9 @@ def _ensure_dvc_installed(self):
 
         ret_code = 1
         self.printer.error("dvc executable is not installed or found")
-        questions = [
-            {
-                'type': 'confirm',
-                'message': 'Should we install dvc [https://dvc.org/] for you right now?\n' +
-                           '  Will install using `pip3 install dvc==2.3.0`',
-                'name': 'install',
-                'default': False,
-            },
-        ]
-        answers = PyInquirer.prompt(questions)
-        if answers["install"]:
+        answer = get_confirm_from_user('Should we install dvc [https://dvc.org/] for you right now?\n' +
+                                       '  Will install using `pip3 install dvc==2.3.0`', False)
```

Comment:
1. If we're still installing `dvc==2.3.0` that limitation should be removed (can create a new unrelated issue for this or fix here)
2. If not, this message should be fixed

Reply:
Let me fix this separately, dont want to merge in this PR

Reply:
#107